website/docs/components/table/partials/code/component-api.md

```diff
@@ -1,38 +1,50 @@
   </C.Property>
   <C.Property @name="model" @type="array">
-    If defined, sets the data source that gets yielded by the `:body` named block.
+    Indicates the data model to be used by the table.
   </C.Property>
   <C.Property @name="columns" @type="array">
-    Use an `array` hash to define your table columns. While `key` and `label` are required, other options include `isSortable`, `align` (for text-alignment), and `width`.
+  Use a `hash` within the array to define each column.
+    <Doc::ComponentApi as |D|>
+      <D.Property @name="key" @type="string">
+      The column’s key; required if the column is sortable.
+      </D.Property>
+      <D.Property @name="label" @type="string" @required="true">
+      The column’s label; supports internationalization.
+      </D.Property>
+      <D.Property @name="isSortable" @type="boolean" @values={{array "false" "true" }} @default="false">
+      If set to `true`, indicates that a column should be sortable.
+      </D.Property>
+      <D.Property @name="sortingFunction" @type="function">
+      Callback function to provide support for a custom callback.
+      </D.Property>
+      <D.Property @name="align" @type="enum" @values={{array "left" "center" "right" }} @default="left">
+        If set, determines the text alignment for the column.
+      </D.Property>
+    </Doc::ComponentApi>
   </C.Property>
   <C.Property @name="sortBy" @type="string">
-    If defined, indicates which column should be pre-sorted when the table is rendered.
+    If defined, the value should be set to the key of the column that should be pre-sorted when the table is rendered.
   </C.Property>
   <C.Property @name="sortOrder" @type="string" @values={{array "asc" "desc" }} @default="asc">
     Use in conjunction with `sortBy`. If defined, indicates which direction the column should be pre-sorted in. If not defined, `asc` is applied by default.
   </C.Property>
-  <C.Property @name="onSort" @type="function">
-    Callback function invoked (if provided) when one of the sortable table headers is clicked, and a sorting is triggered. The function receives the values of `sortBy` and `sortOrder` as arguments.
-  </C.Property>
   <C.Property @name="isStriped" @type="boolean" @values={{array "false" "true" }} @default="false">
     Define on the table invocation. If set to `true`, row striping ("zebra striping") will be applied to the table.
   </C.Property>
-  <C.Property @name="isFixed" @type="boolean" @values={{array "false" "true" }} @default="false">
-    If set to `true`, the `table-display`(CSS) property will be set to `fixed`, which will automatically distribute columns equally based on the total width of the table.
+  <C.Property @name="isFixedLayout" @type="boolean" @values={{array "false" "true" }} @default="false">
+    If set to `true`, the `table-display`(CSS) property will be set to `fixed`, automatically distributing columns based on the total width of the table per the browser algorithm.
```

Comment:
Suggestion: I would simplify this wording and link to the MDN docs for more details.

=> "If set to `true`, the `table-display`(CSS) property will be set to `fixed`."

I would either link part of the above sentence to https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout or else include the link in parenthesis after the sentence.